.circleci/config.yml

```diff
@@ -332,13 +362,13 @@ workflows:
           <<: *tag_filter
           matrix:
             parameters:
-              pythonversion: [3.8.12, 3.9.10, 3.10.2]   
+              pythonversion: [3.8.12, 3.9.10, 3.10.2]
```

Comment:
please revert this and the one on 371 
to keep the changes minimal, and make it easier to do git-blame in the future)